src/pandablocks/responses.py

```diff
@@ -266,6 +266,9 @@ class StartData(Data):
     process: str
     format: str
     sample_bytes: int
+    arm_time: str
+    start_time: str
+    hw_time_offset_ns: str
```

Comment:
This is definitely nicer

Reply:
Default value of `None` doesn't seem to work very well with HDF5. For example, running the following code
```
import h5py

with h5py.File('test.h5', 'w') as f:
    f.attrs['a'] = None
```
results in the following error
```
Traceback (most recent call last):
  File "test_h5py_none.py", line 4, in <module>
    f.attrs['a'] = None
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "/home/dgavrilov/miniconda3/envs/bs-qserver-dev/lib/python3.10/site-packages/h5py/_hl/attrs.py", line 104, in __setitem__
    self.create(name, data=value)
  File "/home/dgavrilov/miniconda3/envs/bs-qserver-dev/lib/python3.10/site-packages/h5py/_hl/attrs.py", line 181, in create
    htype = h5t.py_create(original_dtype, logical=True)
  File "h5py/h5t.pyx", line 1664, in h5py.h5t.py_create
  File "h5py/h5t.pyx", line 1688, in h5py.h5t.py_create
  File "h5py/h5t.pyx", line 1748, in h5py.h5t.py_create
TypeError: Object dtype dtype('O') has no native HDF5 equivalent
```
My idea was to select default values of the same type as normal values so that the types of attributes in HDF5 file are consistent. Another option is not to create the attribute in the HDF5 file if the respective parameter can not be loaded from the header passed by Panda.